src/components/NavigationButton/NavigationButton.tsx

```diff
@@ -1,12 +1,22 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import classes from './NavigationButton.module.scss'
 
 export const NavigationButton: React.FC = () => {
-    return (
-        <div className={classes.navigation}>
-            <button className={classes.buttonNavigation}></button>
-            <button className={classes.buttonNavigation}></button>
+    const [isDate, setIsDate] = useState<string>('');
```

Comment:
`<string>` is not needed here, because will be automatically detected from initial value